third_party/mkl_dnn/mkldnn_acl.BUILD

```diff
@@ -161,7 +161,6 @@ cc_library(
         "src/cpu/aarch64/xbyak_aarch64/xbyak_aarch64",
         "src/cpu/gemm",
     ],
-    linkopts = ["-lgomp"],
```

Comment:
Doesn't the `build_with_mkl_aarch64_openmp` config need to link with this? If so, we probably should do a select. If not, let's remove the whole thing.

Reply:
That's a good point! I did a test OMP build - but I passed `-lgomp` with `--linkopt` on the `bazel build` line. I'll take another look.

Reply:
I've tested the OMP build separately, without any additional `linkopt` passed to the `bazel build` - so I think this is OK.

However, purging all libgomp dependencies might need some changes to the Compute Library build too - currently the OMP scheduler sources are built, but not used for a 'threadpool' build:

https://github.com/tensorflow/tensorflow/blob/e926ffb8089a1d3f31255a7b4e7ed777ff9b157b/third_party/compute_library/BUILD#L16
https://github.com/tensorflow/tensorflow/blob/e926ffb8089a1d3f31255a7b4e7ed777ff9b157b/third_party/compute_library/BUILD#L57

Reply:
Thank you very much! Let's remove it then.

> However, purging all libgomp dependencies might need some changes to the Compute Library build too - currently the OMP scheduler sources are built, but not used for a 'threadpool' build:

It would be great if we can do that at some point. :)